### PR TITLE
minor typo correction: unknown

### PR DESCRIPTION
**Pull request by GitMensch**
additional: reordered checks in brushSystem

### CODE REVIEW

**Review by milianw — CHANGES_REQUESTED** [review]
please split this up into two patches, otherwise this lgtm

File: src/main.cpp [hunk 1]
```diff
@@ -193,6 +193,9 @@ int main(int argc, char** argv)
             << parser.helpText();
         return 1;
     }
+    const auto originalArguments = QCoreApplication::arguments();
```
Comment (milianw) [inline src/main.cpp:196]: why this change?

**Comment by GitMensch** [comment]
Is one PR with two commits fine or so you want separate PRs?

File: src/main.cpp (hunk 1 above)
Comment (GitMensch) [reply src/main.cpp:196]: The idea (I've reverted all but moving that declaration up-front) was to check the arguments before creating the window.
But it's not part of this PR, so I'm just reverting that.

There is now only one thing in this PR, so there's no split useful anymore, is it? If you still want the split: is one commit each (one for the name, the other for the message) enough or do you really like two PRs?

Comment (milianw) [reply src/main.cpp:196]: it's fine to have it in one merge request, but you have to make atomic commits - how many I don't care, but better more than less. each commit gets its own message with a motivational text to set the stage for me as a reviewer why a change is needed.

**Comment by milianw** [comment]
> Is one PR with two commits fine or so you want separate PRs?

one merge request, many commits

**Comment by GitMensch** [comment]
split as requested

**Review by milianw — APPROVED** [review]
